pkgs/development/libraries/spandsp/default.nix

```diff
@@ -1,7 +1,18 @@
-{ lib, stdenv, fetchurl, audiofile, libtiff, buildPackages }:
+{ lib, stdenv, fetchurl, audiofile, libtiff, buildPackages
+, fetchpatch
+, autoreconfHook }:
```

Comment:
I'd complain about the missing empty line after this, but since you already have [another PR](https://github.com/NixOS/nixpkgs/pull/218480) open to reformat this, I won't. :)